e2e/cypress.json

```diff
@@ -1,8 +1,4 @@
 {
     "baseUrl": "http://ckan:5000",
-    "reporter": "junit",
-    "reporterOptions": {
-        "mochaFile": "cypress/results/output.xml"
```

Comment:
Is there any reason we don't want this output file anymore?  If we're worried about not seeing results in the terminal, we could add the `toConsole` parameter as well: https://docs.cypress.io/guides/tooling/reporters#Cypress-configuration-2

Reply:
Oooh, good suggestion. Will rerun locally with those options and see if we get what we want.

Reply:
No, probably not.  So it's okay to leave it as you changed it.  Just wanted to mention 😅

Reply:
No worries. Always glad to spend a bit of effort to figure out what we really want. 

FYI current output to static output.xml only includes the last spec -- by design. https://docs.cypress.io/guides/tooling/reporters#Merging-reports-across-spec-files

Reply:
Good point, it wasn't entirely working before anyway haha..